docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -44,81 +39,186 @@ Specifically, this guide will have you do the following:
 
-```
-wget http://stable.release.core-os.net/amd64-usr/current/coreos_production_iso_image.iso
-```
+## Configure the Kubernetes Master
 
-> You can also download the ISO from the [CoreOS website](https://coreos.com/docs/running-coreos/platforms/iso/).
+1.  Once you've downloaded the ISO image, boot your Kubernetes master from the image.  Once booted, you should be automatically logged in as the `core` user at the terminal. At this point CoreOS is running from the ISO and it hasn't been installed yet.
 
-## Configure the Kubernetes Master
+2.  *On another machine*, download the the [master cloud-config template](https://raw.githubusercontent.com/projectcalico/calico-cni/k8s-1.1-docs/samples/kubernetes/cloud-config/master-config-template.yaml) and save it as `master-config.yaml`.
 
-Once you've downloaded the image, use it to boot your Kubernetes master.  Once booted, you should be automatically logged in as the `core` user.
+3.  Replace the following variables in the `master-config.yaml` file.
 
-*On another machine*, download the `calico-kubernetes` repository, which contains the necessary cloud-config files for this guide, and make a copy of the file `master-config-template.yaml`.
+    - `<SSH_PUBLIC_KEY>`: The public key you will use for SSH access to this server.
 
-```
-wget https://github.com/projectcalico/calico-kubernetes/archive/master.tar.gz
-tar -xvf master.tar.gz
-cp calico-kubernetes-master/config/cloud-config/master-config-template.yaml master-config.yaml
-```
+4.  Copy the edited `master-config.yaml` to your Kubernetes master machine (using a USB stick, for example).
+
+5.  The CoreOS bootable ISO comes with a tool called `coreos-install` which will allow us to install CoreOS and configure the machine using a cloud-config file.  The following command will download and install stable CoreOS using the `master-config.yaml` file we just created for configuration.  Run this on the Kubernetes master.
+
+    > **Warning:** this is a destructive operation that erases disk `sda` on your server.
+
+    ```
+    sudo coreos-install -d /dev/sda -C stable -c master-config.yaml
+    ```
+
+6.  Once complete, restart the server and boot from `/dev/sda` (you may need to remove the ISO image). When it comes back up, you should have SSH access as the `core` user using the public key provided in the `master-config.yaml` file.
+
+### Configure TLS
+
+The master requires the CA certificate, `ca.pem`; its own certificate, `apiserver.pem` and its private key, `apiserver-key.pem` (generated by following the steps linked in the prerequisites section above).
+
+
+1.  Generate the necessary certificates for the master. This [guide for generating Kubernetes TLS Assets](https://coreos.com/kubernetes/docs/latest/openssl.html) explains how to use OpenSSL to generate the required assets.
+
+2.  Send the three files to your master host (using `scp` for example).
```

Comment:
They could.  They also need the ca.pem and ca-key.pem below when filling out node-config.yaml, so I think it is easiest to generate these on a local machine.